docs/tutorials/cox_model/cox_model.ipynb

```diff
@@ -0,0 +1,893 @@
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(12, 61)"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "len(cph.summary), len(model_glum.coef_)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "One might, therefore, wonder if the Poisson approach is competitive in terms of estimation speed. For the dataset here, the Poisson approach, including the data transformation by `survival_split`, turns out to be faster than the Cox model. This speedup is likely aided by tabmat's optimizations for the high-dimensional `week` categorical."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "32.8 ms ± 3.79 ms per loop (mean ± std. dev. of 7 runs, 10 loops each)\n"
+     ]
+    }
+   ],
+   "source": [
+    "%timeit cox_model = CoxPHFitter().fit(df, duration_col='week', event_col='arrest', formula=\"fin + bs(age, df=4) + race + wexp + mar + paro + bs(prio,df=3)\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "18 ms ± 73.4 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)\n"
+     ]
+    }
+   ],
+   "source": [
+    "%timeit model_glum = glum.GeneralizedLinearRegressor(family=\"poisson\",formula=\"arrest ~ fin + bs(age, df=4) + race + wexp + mar + paro + bs(prio,df=3) + C(week)\").fit(survival_split(df, 'week', 'arrest'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Footnotes\n",
+    "<span id=\"fn1\"> 1</span>:\n",
+    "The Cox model assumes that at most one individual has an event at any time (\"no ties\").\n",
+    "\n",
+    "<span id=\"fn2\"> 2</span>:\n",
+    "The log likelihood of an EDM with parameters $\\theta_i$ is\n",
+    "$$\n",
+    "\\sum_i\\frac{y_i\\theta_i - b(\\theta_i)}{\\sigma_i} + \\text{constant}, \n",
+    "$$\n",
+    "with cumulant function $b$ and dispersion parameter $\\sigma_i$. The linear predictor is given by $\\eta_i=g((b'(\\theta_i))$ for link function $g$. The Cox partial log-likelihood cannot be broken down into such a sum over individuals $i$.\n",
+    "\n",
+    "## References\n",
+    "\n",
+    "[1] Carstensen, B. 2023. Who needs the Cox model anyway. December. Available at: http://bendixcarstensen.com/WntCma.pdf.\n",
+    "\n",
+    "[2] Whitehead, J., 1980. Fitting Cox’s regression model to survival data using GLIM. _Journal of the Royal Statistical Society Series C: Applied Statistics_, 29(3), pp.268-275.\n",
+    "\n",
+    "[3] Zhong, C. and Tibshirani, R., 2019. Survival analysis as a classification problem. arXiv preprint arXiv:1909.11171."
```

Comment:
They do use the same dataset-transformation (which they call stacking) idea, but they rely on a binomial model instead of Poisson and say that it is a good enough approximation. So relevant, but I would not say it's the same thing.

Reply:
Right. I removed that reference.